api/nodecontrol/client.go

```diff
@@ -186,3 +187,45 @@ func (c *ControlAPIClient) SetLameDuck(nodeId string) (*nodegen.LameduckResponse
 
 	return &envelope.Data, nil
 }
+
+func (c *ControlAPIClient) MonitorLogs(workloadId, level string) (chan []byte, error) {
```

Comment:
At some point we're going to want to enable people to monitor events and logs from an entire namespace rather than a specific workload id.

Reply:
That is already in the works